### PR TITLE
Update testing 

### PR DESCRIPTION
**Pull request by croixk**
- Add testing for listing service, search feature

### CODE REVIEW

**Review by kevingloss — APPROVED** [review]
Looks good!

File: spec/services/listing_service_spec.rb [hunk 1]
```diff
@@ -1,98 +1,60 @@
...
 
-RSpec.describe ListingService do
-
-  it 'gets listing based on given id', :vcr do
-    listing = ListingService.get_listing_by_id(1)
-    expect(listing).to be_a(Hash)
-    expect(listing[:data][:attributes][:year]).to be_an(Integer)
-    expect(listing[:data][:attributes][:make]).to be_a(String)
-    expect(listing[:data][:attributes][:model]).to be_a(String)
+RSpec.describe 'listing service' do
```
Comment (Eldridge-Turambi) [inline spec/services/listing_service_spec.rb:3]: More description here. Looks good

**Review by Eldridge-Turambi — APPROVED** [review]

File: spec/services/listing_service_spec.rb [hunk 2]
```diff
@@ -1,98 +1,60 @@
...
+
+      # let!(:make_response) { ListingService.get_makes }
+      # let!(:json) { JSON.parse(make_response.body, symbolize_names: true) }
+
+
+      xit "returns correct https status " do
+        expect(make_response).to have_http_status(200)
+      end
+
+      xit "returns the expected json" do
```
Comment (LelandCurtis) [inline spec/services/listing_service_spec.rb:17]: we can delete this now.

File: spec/services/listing_service_spec.rb [hunk 3]
```diff
@@ -1,98 +1,60 @@
...
+
+    context 'class methods', :vcr do
+      describe 'conn' do
+        it 'can return listings hash' do
+          listing = ListingService.conn
+          expect(listing).to be_a(Faraday::Connection)
+        end
+      end
+
+      describe '.get listings' do
```
Comment (LelandCurtis) [inline spec/services/listing_service_spec.rb:33]: we should add a test for the https response (200)

**Pull request merged**